2018-01-18 17:46:46,624 ERROR 19726 [-/127.0.0.1/-/4ms GET /] nodejs.TypeError: this.config.keys.split is not a function
    at Application.get keys [as keys] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg/lib/application.js:242:37)
    at Object.get cookies [as cookies] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg/app/extend/context.js:18:66)
    at Object.get [egg-security#CSRF_SECRET] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/app/extend/context.js:77:33)
    at Object.ensureCsrfSecret (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/app/extend/context.js:88:13)
    at csrf (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/lib/middlewares/csrf.js:14:9)
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:42:32)
    at /Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:42:32)
    at next (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:43:18)
    at session (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-session/index.js:41:13)

pid: 19726
hostname: fangshideMacBook-Pro.local

2018-01-18 17:47:10,030 ERROR 19726 [-/127.0.0.1/-/1ms GET /] nodejs.TypeError: this.config.keys.split is not a function
    at Application.get keys [as keys] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg/lib/application.js:242:37)
    at Object.get cookies [as cookies] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg/app/extend/context.js:18:66)
    at Object.get [egg-security#CSRF_SECRET] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/app/extend/context.js:77:33)
    at Object.ensureCsrfSecret (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/app/extend/context.js:88:13)
    at csrf (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-security/lib/middlewares/csrf.js:14:9)
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:42:32)
    at /Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:42:32)
    at next (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-compose/index.js:43:18)
    at session (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-session/index.js:41:13)

pid: 19726
hostname: fangshideMacBook-Pro.local

2018-01-18 19:09:11,258 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 1589798ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-18 21:12:12,428 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 300191ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-18 23:13:50,580 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3257348ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 00:20:28,305 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3957705ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 01:20:31,898 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3563571ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 02:13:58,450 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3186536ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 03:13:53,040 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3554572ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 04:23:20,626 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 4127573ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 05:13:56,802 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 2996161ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 06:14:21,110 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3584281ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 07:19:38,715 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3877584ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 08:15:17,260 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3318535ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 09:15:32,852 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3575569ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 10:14:04,385 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 3471512ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 10:20:03,961 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 319561ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 11:18:03,851 ERROR 19901 nodejs.ClusterClientNoResponseError: client no response in 279221ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 19901
hostname: fangshideMacBook-Pro.local

2018-01-19 12:09:12,885 ERROR 23597 [-/127.0.0.1/-/3ms GET /news] nodejs.AssertionError [ERR_ASSERTION]: Can't find ViewEngine "nunjucks"
    at ContextView.[contextView#getViewEngine] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:87:5)
    at ContextView.[contextView#render] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:68:39)
    at <anonymous>
generatedMessage: false
name: 'AssertionError [ERR_ASSERTION]'
code: 'ERR_ASSERTION'
actual: undefined
expected: true
operator: '=='
pid: 23597
hostname: fangshideMacBook-Pro.local

2018-01-19 12:12:51,200 ERROR 23597 [-/127.0.0.1/-/1ms GET /news] nodejs.AssertionError [ERR_ASSERTION]: Can't find ViewEngine "nunjucks"
    at ContextView.[contextView#getViewEngine] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:87:5)
    at ContextView.[contextView#render] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:68:39)
    at <anonymous>
generatedMessage: false
name: 'AssertionError [ERR_ASSERTION]'
code: 'ERR_ASSERTION'
actual: undefined
expected: true
operator: '=='
pid: 23597
hostname: fangshideMacBook-Pro.local

2018-01-19 12:21:15,489 ERROR 23673 [-/127.0.0.1/-/8ms GET /news] nodejs.AssertionError [ERR_ASSERTION]: Can't find ViewEngine "nunjucks"
    at ContextView.[contextView#getViewEngine] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:87:5)
    at ContextView.[contextView#render] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-view/lib/context_view.js:68:39)
    at <anonymous>
generatedMessage: false
name: 'AssertionError [ERR_ASSERTION]'
code: 'ERR_ASSERTION'
actual: undefined
expected: true
operator: '=='
pid: 23673
hostname: fangshideMacBook-Pro.local

2018-01-19 13:11:27,950 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 1849758ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 19:37:26,987 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 130866ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 20:46:56,218 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 4149221ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 21:07:28,075 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 1211845ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 21:22:20,615 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 872529ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 21:38:00,509 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 919618ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 21:50:08,742 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 708146ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:00:02,605 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 573853ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:09:57,143 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 574524ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:19:51,695 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 574545ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:29:45,235 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 573532ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:39:40,119 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 574869ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:49:36,536 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576377ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 22:59:32,604 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576003ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:09:29,490 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576720ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:19:25,726 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576156ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:29:23,049 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577241ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:39:19,327 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576193ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:49:16,563 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577152ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-19 23:59:13,324 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576686ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:09:10,124 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576716ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:19:07,753 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577538ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:29:04,309 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576478ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:39:01,033 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576647ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:48:57,343 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576244ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 00:58:54,427 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577042ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:08:51,297 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576828ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:18:47,591 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576251ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:28:44,538 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576906ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:38:40,793 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576164ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:48:37,810 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576977ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 01:58:34,053 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576186ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:08:30,525 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576387ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:18:27,558 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576946ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:28:24,737 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577063ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:38:21,741 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576930ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:48:17,727 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 575894ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 02:58:15,039 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577231ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:08:10,656 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 575529ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:18:07,821 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577077ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:28:05,033 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577126ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:38:02,166 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577052ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:47:58,620 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576365ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 03:57:55,166 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576461ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 04:07:52,503 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577247ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 04:17:49,119 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576540ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 04:23:41,699 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 332505ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 04:33:38,173 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576388ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 04:54:12,058 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 1213797ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 05:09:43,272 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 911136ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 05:21:52,533 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 709172ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 05:31:49,881 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577257ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 05:41:46,715 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576749ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 05:51:43,619 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576818ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:01:40,580 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576806ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:11:37,010 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576279ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:21:33,791 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576722ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:31:30,757 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576921ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:41:27,086 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576289ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 06:51:23,509 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576389ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 07:01:20,988 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577417ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 07:11:17,319 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576252ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 07:21:13,451 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 576066ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 07:31:10,506 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 577007ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-20 07:36:32,992 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 302425ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-22 10:00:43,129 ERROR 23596 nodejs.ClusterClientNoResponseError: client no response in 181450070ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 23596
hostname: fangshideMacBook-Pro.local

2018-01-22 10:38:47,874 ERROR 30623 [-/127.0.0.1/-/6ms GET /news] nodejs.TypeError: Cannot destructure property `serverUrl` of 'undefined' or 'null'.
    at NewsService.list (/Users/mine/FE/sohi/learning/egg/epEgg/app/service/news.js:5:47)
    at NewsController.list (/Users/mine/FE/sohi/learning/egg/epEgg/app/controller/news.js:24:45)
    at Object.callFn (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-core/lib/loader/mixin/controller.js:84:20)
    at Object.callFn (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-core/lib/utils/index.js:44:21)
    at wrappedController (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/egg-core/lib/utils/router.js:322:18)
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-router/lib/router.js:345:16
    at dispatch (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)

pid: 30623
hostname: fangshideMacBook-Pro.local

2018-01-22 10:40:09,426 ERROR 30645 [-/undefined/-/5023ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":2,"createSocketErrorCount":0,"closeSocketCount":1,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{"hacker-news.firebaseio.com:443::::::::::":1},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 1
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5015, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
headerSent: true
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:40:10,841 ERROR 30645 [-/undefined/-/5002ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":3,"createSocketErrorCount":0,"closeSocketCount":2,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{"hacker-news.firebaseio.com:443::::::::::":1},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 2
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5001, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
headerSent: true
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:40:14,999 ERROR 30645 [-/127.0.0.1/-/5001ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":3,"createSocketErrorCount":0,"closeSocketCount":3,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 3
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5001, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:56:42,907 ERROR 30645 [-/undefined/-/5004ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":5,"createSocketErrorCount":0,"closeSocketCount":4,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{"hacker-news.firebaseio.com:443::::::::::":1},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 4
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5004, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
headerSent: true
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:56:44,080 ERROR 30645 [-/127.0.0.1/-/5004ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":5,"createSocketErrorCount":0,"closeSocketCount":5,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 5
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5002, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:56:58,156 ERROR 30645 [-/undefined/-/5004ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":6,"createSocketErrorCount":0,"closeSocketCount":6,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 6
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5004, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
headerSent: true
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:57:13,183 ERROR 30645 [-/undefined/-/5006ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":8,"createSocketErrorCount":0,"closeSocketCount":7,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{"hacker-news.firebaseio.com:443::::::::::":1},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 7
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5005, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
headerSent: true
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 10:57:17,162 ERROR 30645 [-/127.0.0.1/-/5004ms GET /news] nodejs.ConnectionTimeoutError: Connect timeout for 5000ms, GET https://hacker-news.firebaseio.com/v0/topstories.json -2 (connected: false, keepalive socket: false, agent status: {"createSocketCount":8,"createSocketErrorCount":0,"closeSocketCount":8,"errorSocketCount":0,"timeoutSocketCount":0,"requestCount":0,"freeSockets":{},"sockets":{},"requests":{}})
headers: {}
headers: {}
    at Timeout._onTimeout (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/urllib/lib/urllib.js:735:15)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ConnectionTimeoutError'
requestId: 8
data: undefined
path: '/v0/topstories.json?orderBy=%22%24key%22&startAt=%220%22&endAt=%224%22'
status: -2
headers: {}
res: { status: -2, statusCode: -2, headers: {}, size: 0, aborted: false, rt: 5002, keepAliveSocket: false, data: undefined, requestUrls: [ 'https://hacker-news.firebaseio.com/v0/topstories.json' ], timing: null, remoteAddress: '35.201.97.85', remotePort: '' }
pid: 30645
hostname: fangshideMacBook-Pro.local

2018-01-22 12:39:11,959 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 1477283ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-22 22:06:38,613 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3525893ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-22 22:15:41,024 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 482393ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-22 22:25:10,689 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 269536ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-22 23:07:34,248 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 2503538ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 00:05:07,539 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3413271ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 00:06:45,023 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 77468ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 01:06:12,283 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3527245ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 02:08:14,852 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3682545ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 03:02:06,417 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3211555ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 04:07:33,990 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3887557ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 05:08:58,585 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3644577ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 06:02:36,170 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3177566ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 07:04:18,752 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3662565ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 08:08:04,580 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3805814ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 09:07:31,125 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 3526529ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 09:17:44,644 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 573499ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 10:05:11,575 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 2826921ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 10:08:01,381 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 89791ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 10:21:57,646 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 816254ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 11:20:28,205 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 307335ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

2018-01-23 13:05:05,833 ERROR 30622 nodejs.ClusterClientNoResponseError: client no response in 1796578ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mine/FE/sohi/learning/egg/epEgg/node_modules/cluster-client/lib/leader.js:73:23)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
name: 'ClusterClientNoResponseError'
pid: 30622
hostname: fangshideMacBook-Pro.local

